jerry-core/debugger/debugger.c

```diff
@@ -160,7 +160,7 @@ jerry_debugger_send_eval (const lit_utf8_byte_t *eval_string_p, /**< evaluated s
   }
 
   ecma_value_t message = result;
-  uint8_t type = JERRY_DEBUGGER_EVAL_RESULT;
+  uint8_t type = JERRY_DEBUGGER_EVAL_OK;
```

Comment:
Does this variable changes its value? It seems a constant. If it is, perhaps we could simply remove it.